containerbuddy/signals.go

```diff
@@ -55,33 +56,33 @@ func terminate(config *Config) {
 	}
 	if config.StopTimeout > 0 {
 		if err := cmd.Process.Signal(syscall.SIGTERM); err != nil {
-			log.Printf("Error sending SIGTERM to application: %s\n", err)
+			log.Warnf("Error sending SIGTERM to application: %s\n", err)
 		} else {
 			time.AfterFunc(time.Duration(config.StopTimeout)*time.Second, func() {
-				log.Printf("Killing Process %#v\n", cmd.Process)
+				log.Infof("Killing Process %#v\n", cmd.Process)
 				cmd.Process.Kill()
 			})
 			return
 		}
 	}
-	log.Printf("Killing Process %#v\n", config.Command.Process)
+	log.Infof("Killing Process %#v\n", config.Command.Process)
 	cmd.Process.Kill()
 }
 
 func reloadConfig(config *Config) *Config {
 	signalLock.Lock()
 	defer signalLock.Unlock()
-	log.Printf("Reloading configuration.\n")
+	log.Infof("Reloading configuration.\n")
 	newConfig, err := loadConfig()
 	if err != nil {
-		log.Printf("Could not reload config: %v\n", err)
+		log.Warnf("Could not reload config: %v\n", err)
```

Comment:
I'm going to nitpick and suggest this could be at the Error level, given that if this error is thrown the behavior of Containerbuddy will start to converge from what the user expects.